{{cookiecutter.repostory_name}}/noxfile.py

```diff
@@ -8,25 +8,31 @@
 
 import nox
 
-os.environ["PDM_IGNORE_SAVED_PYTHON"] = "1"
-
 CI = os.environ.get("CI") is not None
 
 ROOT = Path(".")
 PYTHON_VERSIONS = ["3.11"]
 PYTHON_DEFAULT_VERSION = PYTHON_VERSIONS[-1]
 APP_ROOT = ROOT / "app" / "src"
 
-nox.options.default_venv_backend = "venv"
+nox.options.default_venv_backend = "uv"
 nox.options.stop_on_first_error = True
 nox.options.reuse_existing_virtualenvs = not CI
 
 
 def install(session: nox.Session, *args):
     groups = []
-    for group in args:
-        groups.extend(["--group", group])
-    session.run("pdm", "install", "--check", *groups, external=True)
+    for arg in args:
+        groups.extend(["--group", arg])
+
+    uv_env = getattr(session.virtualenv, "location", os.getenv("VIRTUAL_ENV"))
```

Comment:
what's the use case for virtualenv not being set in the session object but being given by an env var?